packages/integration-tests/cmr.js

```diff
@@ -303,19 +265,8 @@ async function getOnlineResourcesECHO10(cmrLink) {
     Description: "Download MOD09GQ.A0794505._4kqJd.006.9457902462263.hdf",
     Type: "GET DATA" }
  */
-async function getOnlineResourcesUMMG(cmrLink) {
-  const response = await got.get(cmrLink);
-
-  if (response.statusCode !== 200) {
-    return null;
-  }
-
-  const body = JSON.parse(response.body);
-
-  const links = body.items.map((item) => item.umm.RelatedUrls);
-
-  // Links is a list of a list, so flatten to be one list
-  return [].concat(...links);
+async function getOnlineResourcesUMMG(_cmrLink) {
+  return await Promise.resolve([{ href: 'dummy', URL: 'dummy.com' }]);
```

Comment:
Only need to fake getOnlineResources.

Reply:
I suggest to keep all the old methods, and name the new ones xxxFake, and export the methods like,
getOnlineResources: getOnlineResourcesFake
It's very easy to change back when cmr is fixed.